live_pipeline/get_reviews.py

```diff
@@ -53,6 +58,8 @@ def get_all_reviews(game_ids: list[int]) -> None:
 
         for page in range(int(reviews_info[-1]["number_of_total_reviews"]/100)+2):
             api_response = get_reviews_for_game(game, cursor_list[page])
+            if "error" in list(api_response.keys()):
+                continue
             cursor = api_response["next_cursor"]
             page_reviews = api_response["reviews"]
             if not page_reviews or cursor in cursor_list:
```

Comment:
It looks like what you want this line to do is break if either of page reviews and cursor are not in cursor_list
if you want this behaviour
```suggestion
            if (not page_reviews in cursor_list) or (not cursor in cursor_list):
```

I think what it's actually doing is:
if (not page_reviews) or (cursor in cursor_list):

it'll break if cursor is in cursor_list, or if page_reviews has a falsy value (e.g. empty dict)
if you want this behaviour I'd break this line into two separate if statements for clarity

Reply:
The continue statement is there to get other reviews if it stops on some so we don't not receive none. The additional "not" will be added - good spot! Thank you

Reply:
Ah I realised that the cursor should actually not have the extra "not". This is because when API gets no reviews the page_reviews variable goes to [] which is a false value in Python. The cursor repeats when there are no more values, which means we should stop, which is why we check if it was already in the list of cursors. This is why there's a break and why there is not an extra "not".